active_directory/README.md

```diff
@@ -32,6 +32,42 @@ If installing the Datadog Agent on a domain environment, see [the installation r
 
 See [metadata.csv][8] for a list of metrics provided by this integration.
 
+The integration collects metrics from the following Windows Performance Objects:
+
+- **NTDS**: Core Active Directory metrics including replication, LDAP operations, and directory service threads
+- **Netlogon**: Authentication performance metrics including semaphore statistics for monitoring authentication bottlenecks
+- **Security System-Wide Statistics**: Authentication protocol usage metrics (NTLM vs Kerberos)
+- **DHCP Server**: DHCP failover and binding update metrics (when DHCP Server role is installed)
+- **DFS Replicated Folders**: DFS replication health, conflicts, and staging metrics (when DFSR role is installed)
+  - Note: Metrics are tagged with `replication_group` containing the DFS replication group name
+
+#### Netlogon Metrics
+
+The Netlogon metrics help monitor authentication performance and identify bottlenecks in domain controller authentication processing:
+
+- `active_directory.netlogon.semaphore_waiters`: Number of threads waiting for the authentication semaphore
+- `active_directory.netlogon.semaphore_holders`: Number of threads currently holding the semaphore
+- `active_directory.netlogon.semaphore_acquires`: Total number of semaphore acquisitions
+- `active_directory.netlogon.semaphore_timeouts`: Number of timeouts waiting for the semaphore
+- `active_directory.netlogon.semaphore_hold_time`: Average time (in seconds) the semaphore is held
```

Comment:
that kind of interesting. It looks like our documentation (see https://docs.datadoghq.com/integrations/active-directory/) automatically expand metadata.csv